deps/deps.jl

```diff
@@ -0,0 +1,11 @@
+# This is an auto-generated file; do not edit
+
+# Pre-hooks
+
+# Macro to load a library
+macro checked_lib(libname, path)
```

Comment:
Where does this get used?  Is it somewhere automagically required by BinDeps, or should we be doing an extra include inside "Proj4.jl" ?